ui/app/templates/partials/secret-list/transform-transformation-item.hbs

```diff
@@ -0,0 +1,62 @@
+{{#linked-block
+  (concat
+    "vault.cluster.secrets.backend."
+    "show"
+  )
+  item.id
+  class="list-item-row"
+  data-test-secret-link=item.id
+  encode=true
+  queryParams=(secret-query-params backendModel.type)
+}}
+  <div class="columns is-mobile">
+  <div class="column is-10">
+    <SecretLink
+      @mode="show"
+      @secret={{item.id}}
+      @queryParams={{if (eq backendModel.type "transform") (query-params tab="actions") ""}}
+      @class="has-text-black has-text-weight-semibold">
+      <Icon
+        @glyph='file-outline'
+        @class="has-text-grey-light"/>
+      {{if (eq item.id ' ') '(self)' (or item.keyWithoutParent item.id)}}
+    </SecretLink>
+  </div>
+  <div class="column has-text-right">
+    <PopupMenu name="secret-menu">
+      <nav class="menu">
+        <ul class="menu-list">
+          {{#if (or item.versionPath.isLoading item.secretPath.isLoading)}}
+            <li class="action">
+              <button disabled type="button" class="link button is-loading is-transparent">
+                loading
+              </button>
+            </li>
+          {{else}}
+            {{#if item.updatePath.canRead}}
```

Comment:
I think you followed the general item pattern here so not a blocker, but if we check `canRead` for the details dropdown link, should we check it for the main item link as well?